auth-server/.eslintrc.json

```diff
@@ -6,34 +6,44 @@
     "extends": [
         "eslint:recommended",
         "plugin:@typescript-eslint/eslint-recommended",
-        "plugin:security/recommended"
+        "plugin:@typescript-eslint/recommended",
+        "plugin:@typescript-eslint/recommended-requiring-type-checking",
+        "plugin:import/typescript",
+        "semistandard"
     ],
     "globals": {
         "Atomics": "readonly",
         "SharedArrayBuffer": "readonly"
     },
     "parser": "@typescript-eslint/parser",
     "parserOptions": {
-        "ecmaVersion": 2018,
-        "sourceType": "module"
-    },
+        "project": ["./tsconfig.json"]
+      },
     "plugins": [
         "@typescript-eslint",
         "header",
-        "security"
+        "security",
+        "promise",
+        "simple-import-sort"
     ],
     "rules": {
+        "@typescript-eslint/camelcase": "off",
```

Comment:
I had to add this because we have many `camel_case` vars.. and tbh that's not a big deal. Let's just try to avoid those in the future.